at_secondary/at_secondary_server/lib/src/verb/handler/config_verb_handler.dart

```diff
@@ -83,6 +83,43 @@ class ConfigVerbHandler extends AbstractVerbHandler {
           result = 'unknown operation';
           break;
       }
+
+      switch (setOperation) {
+        case 'set':
+          if (AtSecondaryConfig.testingMode &&
+              ModifiableConfigs.values.contains(verbParams[CONFIG_NAME])) {
+            AtSecondaryConfig.broadcastConfigChange(
+                ModifiableConfigs.values.byName(verbParams[CONFIG_NAME]!),
+                int.parse(verbParams[CONFIG_VALUE]!));
+            result = 'ok';
+          } else {
+            result = 'please enter valid config name';
+          }
+          break;
+        case 'reset':
+          if (AtSecondaryConfig.testingMode &&
+              ModifiableConfigs.values.contains(verbParams[CONFIG_NAME])) {
+            AtSecondaryConfig.broadcastConfigChange(
+                ModifiableConfigs.values.byName(verbParams[CONFIG_NAME]!), null,
+                isReset: true);
+            result = 'ok';
+          } else {
+            if (AtSecondaryConfig.testingMode &&
+                ModifiableConfigs.values.contains(verbParams[CONFIG_NAME])) {
+              result = AtSecondaryConfig.getLatestConfigValue(
+                  ModifiableConfigs.values.byName(verbParams[CONFIG_NAME]!));
+            } else {
+              result = 'null';
+            }
+          }
+          break;
+        case 'print':
+          result = 'print operation';
```

Comment:
Is this work in progress?

Reply:
i might have missed pushing the latest changes sir. I will push the print implementation along with the review changes suggested.